configuration/items.md

```diff
@@ -118,7 +118,20 @@ In the example above, if you move the Slider widget to 60%, move the Switch to O
 The Item name is the unique identified of the Item.
 The name should only consist of letters, numbers and the underscore character.
 Spaces and special characters cannot be used.
-A good naming schema can be advised.
+
+**Style Guide:**
+It is advised to follow a predefined naming scheme.
+The officially recommended scheme is used throughout this documentation and builds on a hierarchy idea.
+Floor or room names can be abbreviated to reduce overall name length.
+
+Every word of the Item name starts with a uppercase letter.
+Words are separated by an underscore character, except for logically belonging words.
+
+Examples:
+
+- "`BathRoom_WaschingMachine_Power`" for the used power of a washing machine in a single floor apartment bathroom
+- "`GF_LR_Heating_Setpoint`" for the Groundfloor Livingroom set temperature
```

Comment:
How about
"`GF_Living_Heating_Setpoint`" for the Ground floor Living room set temperature
?

I would like to avoid abbreviating specific rooms.

Reply:
Not sure, the example is there to **showcase** abbreviations and the user doesn't have to do it the same way.